spec/03-plugins/26-prometheus/09-wasmx_spec.lua

```diff
@@ -0,0 +1,298 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson"
+
+
+local TEST_NAME_HEADER = "X-PW-Test"
+local TESTS_FILTER_FILE = helpers.test_conf.wasm_filters_path .. "/tests.wasm"
+
+
+local status_api_port = helpers.get_available_port()
```

Comment:
I suggest moving this into `lazy_setup()` so that this does not run until it is truly required.